t/14-isotovideo.t

```diff
@@ -51,6 +52,20 @@ subtest 'get the version number' => sub {
     ok(!-e bmwqemu::STATE_FILE, 'no state file was written');
 };
 
+subtest 'color output can be configured via the command-line' => sub {
+    chdir($pool_dir);
+    unlink('vars.json') if -e 'vars.json';
+    my $out = stderr_from { isotovideo(opts => "--color=yes casedir=$data_dir/tests schedule=foo,bar/baz _exit_after_schedule=1") };
+    isnt($out, colorstrip($out), 'logs use colors when requested');
+    $out = stderr_from { isotovideo(opts => "--color=no casedir=$data_dir/tests schedule=foo,bar/baz _exit_after_schedule=1") };
+    is($out, colorstrip($out), 'no colors in logs');
+  TODO: {
+        local $TODO = 'STDERR and STDOUT are always absent here';
```

Comment:
I doubt STDERR/STDOPUT are completely absent here. They're just not referring to a terminal. Not sure whether we would ever add such a test. So maybe it would be better to remove the whole TODO block.

Reply:
That's what I mean by absent. I don't know how exactly it is implemented.

The reason to have it is coverage of the code path.